### PR TITLE
Added GCS static Website

### PR DESCRIPTION
**Pull request by rayman-g**

### CODE REVIEW

File: gcs/cloud-storage/template/template.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    name: cloud-storage
+    type: tfmodule
+    providers: *provider_gcp
+    source: "github.com/terraform-google-modules/terraform-google-cloud-storage.git?ref=v4.0.1"
+    inputs:
+      names:
+        - {{ .variables.name }}
+      randomize_suffix: true
+      project_id: {{ .variables.project }}
+      location: "EU"
```
Comment (voatsap) [inline gcs/cloud-storage/template/template.yaml:19]: Hardcoded values

File: gcs/cloud-storage/template/template.yaml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+          protocol: "HTTP"
+          port: 80
+          port_name: "http"
+          timeout_sec: 10
+          enable_cdn: false
+          groups: [] 
+          health_check:
+            request_path: "/"
+            port: 80
+          log_config:
```
Comment (voatsap) [inline gcs/cloud-storage/template/template.yaml:71]: Can you set only required values, if this on is not required disable it.

File: gcs/cloud-storage/project.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+name: dev
+kind: Project
+backend: gcs-backend
+variables:
+  organization: my-organization
```
Comment (voatsap) [inline gcs/cloud-storage/project.yaml:5]: remove not used in stackTemplate

File: gcs/cloud-storage/stack.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+name: cloud-storage
+template: ./template/
+kind: Stack
+backend: gcs-backend
+variables:
+  name: {{ .project.variables.name }}
+  region: {{ .project.variables.region }}
+  project: {{ .project.variables.project }}
+  organization: {{ .project.variables.organization }}
```
Comment (voatsap) [inline gcs/cloud-storage/stack.yaml:9]: same for remove

File: gcs/cloud-storage/backend.yaml [hunk 5]
```diff
@@ -0,0 +1,7 @@
+name: gcs-backend
+kind: Backend
+provider: gcs
+spec:
+  project: {{ .project.variables.project }}
```
Comment (voatsap) [inline gcs/cloud-storage/backend.yaml:5]: Remove if not required

File: gcs/cloud-storage/project.yaml [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+  project: cdev-demo
```
Comment (voatsap) [inline gcs/cloud-storage/project.yaml:6]: remove if not required

File: gcs/cloud-storage/template/template.yaml [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+units:
+  -
+    name: cloud-storage
+    type: tfmodule
+    providers: *provider_gcp
+    source: "github.com/terraform-google-modules/terraform-google-cloud-storage.git?ref=v4.0.1"
+    inputs:
+      names:
+        - {{ .variables.name }}
+      randomize_suffix: true
```
Comment (voatsap) [inline gcs/cloud-storage/template/template.yaml:17]: remove if could be defaulted

**Review by voatsap — CHANGES_REQUESTED** [review]

File: gcs/cloud-storage/template/template.yaml (hunk 7 above)
Comment (rayman-g) [reply gcs/cloud-storage/template/template.yaml:17]: To avoid name collisions with existing buckets we should enable this option to randomize bucket name.

**Review by voatsap — APPROVED** [review]
Ok lets merge

**Review by voatsap — APPROVED** [review]
Ok lets merge

**Pull request merged**